Examples/topics/README.md

```diff
@@ -0,0 +1,15 @@
+# Momento Topics Example
+
+## Prerequisites
+
+- A Momento API key is required, you can generate one using the [Momento Console](https://console.gomomento.com). 
+- A Momento Cache, which you can also create using the [Momento Console](https://console.gomomento.com). (The example expects a cache named `my-cache` to exist.)
```

Comment:
Can you just add a quick note that they can create the cache in the console as well?